cmd/gophermart/internal/accrual/request.go

```diff
@@ -0,0 +1,52 @@
+package accrual
+
+import (
+	"encoding/json"
+	"github.com/SergeyGushan/gophermart/cmd/gophermart/config"
+	"io"
+	"net/http"
+)
+
+type ResponseItemStructure struct {
+	Order   string  `json:"order"`
+	Status  string  `json:"status"`
+	Accrual float64 `json:"accrual"`
+}
+
+type ResponseStructure struct {
+	Code int
+	Item ResponseItemStructure
+}
+
+func Request(orderID string) (ResponseStructure, error) {
```

Comment:
```suggestion
func GetOrderAccrual(orderID string) (ResponseStructure, error) {
```
реквестов может быть много, нужно явно описать что делает этот